pkgs/top-level/all-packages.nix

```
@@ -11853,6 +11853,7 @@ in
   openbabel = callPackage ../development/libraries/openbabel { };
 
   opencascade = callPackage ../development/libraries/opencascade { };
+  opencascade_7_3_0 = callPackage ../development/libraries/opencascade-occt { };
```

If occt is different from oce, then it shouldn't have the same attr prefix.
```suggestion
  opencascade-occt_7_3_0 = callPackage ../development/libraries/opencascade-occt { };
```